apps/fly-my-rockets/src/app/rockets/rocket-show/rocket-show.component.ts

```diff
@@ -161,4 +171,10 @@ export class RocketShowComponent {
         this.rocketUpdated$.next();
       });
   }
+
+  private deleteRocket(): void {
+    this.router.navigate(['/rockets'], { replaceUrl: true }).then(() => {
+      this.rocketService.deleteRocket(this.rocketId).subscribe();
+    });
```

Comment:
Deleting the rocket after navigating avoids a console error about the rocket not existing before navigating.